docs/components/modeler/desktop-modeler/lint-rules/lint-rules.md

```diff
@@ -0,0 +1,82 @@
+---
+id: lint-rules
+title: Lint rules
+description: "Learn how to contribute additional validation (lint) rules to the Desktop Modeler through lint rule plug-ins."
+---
+
+The Desktop Modeler uses [bpmnlint](https://github.com/bpmn-io/bpmnlint) to validate BPMN diagrams. Through defined modeler extension points you add plug-in additional lint rules or configure/disable existing rules.
```

Comment:
Typo "you add plug-in additional..."

Reply:
Also, better to highlight the value of this feature first (allows to add your own validation rules to ensure processes follow blabla...) before diving into the technical details

Reply:
Reworded this in my latest commit -- is this what you mean?

Reply:
@philippfromme @christinaausley more like explaining what those rules are for—why should a user care about it? Maybe also adding a screenshot of how a custom rule will appear in the problems panel, making it easier to correlate for the reader

Reply:
I was also thinking about adding a screenshot to sell it better. Will make adjustments. I'll also try and discuss this again with @nikku.